flags.go

```diff
@@ -381,7 +381,7 @@ func newApp() (app *cli.App) {
 			cli.StringFlag{
 				Name:  MetadataPrefetchOnMountFlag,
```

Comment:
Do we want to change the flag name to experimentalMetadataPrefetchOnMountFlag? This is what has been done for other flags to mark them as experimental.

Reply:
This is just an internal variable name, which makes no difference to the end-user. So, I don't think it's worth making changing at many places to rename this variable.